File: fundamentos-de-programacao/list-3/exercise-13.py
alunos = int(input("Quantidade de Alunos: "))

print("Caso tenha adicionado todas as notas de determinado aluno, digite -1")
for i in range(1, alunos + 1):
  total = 0
  divisao = 0

  lastNum = 0
  while (lastNum != -1):
    lastNum = int(input(f"Nota do {i}º aluno: "))
    if (lastNum >= 0):
      total += lastNum
      divisao += 1
  
  if (divisao != 0):
    print(f"Média do {1}º aluno: {total / divisao}")

File: fundamentals-of-programming/bimester-2/list-1/exercise-13.c
#include <stdio.h>
#include <stdlib.h>

main() {
  int alunos, soma = 0, nota, cont;

  printf("Digite a quantidade de alunos: ");
  scanf("%d", &alunos);

  for (cont = 1; cont <= alunos; cont++) {
    printf("Digite a nota do %dº aluno: ", cont);
    scanf("%d", &nota);

    soma += nota;
  }

  printf("Média das notas: %f", (float)soma / alunos);
}